apisix/discovery/nacos/init.lua

```diff
@@ -361,8 +361,9 @@ function _M.nodes(service_name, discovery_args)
         waiting_time = waiting_time - step
     end
 
-    if not applications or not applications[namespace_id] or not applications[namespace_id][group_name] then
-        return nil
+    if not applications or not applications[namespace_id]
+      or not applications[namespace_id][group_name] then
+      return nil
```

Comment:
four spaces are needed

Reply:
if not applications or not applications[namespace_id]
        or not applications[namespace_id][group_name] 
    then
        return nil

Reply:
done